django_prices/models.py

```diff
@@ -25,41 +25,41 @@ def __set__(self, obj, value):
             obj.__dict__[self.field.name] = self.field.to_python(value)
 
 
-class PriceField(models.DecimalField):
+class AmountField(models.DecimalField):
 
-    description = 'A field that stores a price.'
+    description = 'A field that stores an amount'
 
     def __init__(self, verbose_name=None, currency=None, **kwargs):
         self.currency = currency
-        super(PriceField, self).__init__(verbose_name, **kwargs)
+        super(AmountField, self).__init__(verbose_name, **kwargs)
 
     def contribute_to_class(self, cls, name, **kwargs):
-        super(PriceField, self).contribute_to_class(cls, name, **kwargs)
+        super(AmountField, self).contribute_to_class(cls, name, **kwargs)
         setattr(cls, self.name, Creator(self))
```

Comment:
What if we were to drop the `Creator` altogether? They've been deprecated for a long time and coercing types at model level seems like a bad idea.

Reply:
@patrys do you have any link to some kind of documentation? I am trying to find sth about this but with no luck

Reply:
https://docs.djangoproject.com/en/1.8/howto/custom-model-fields/#converting-values-to-python-objects

Reply:
@patrys Ok so i need a quick clarification. If this kind of initialization
```python
def test_instance_values():
    instance = Model(price_net=25)
    assert instance.price.net.value == 25```
Should be supported by new api?
According to:
https://docs.djangoproject.com/en/1.10/releases/1.8/#subfieldbase
https://code.djangoproject.com/ticket/26807
it needs reimplementation of Creator class.
Am i missing something?

Reply:
Ok so we don't want to support casting to Amount on this kind of Model initialization.